plugins/lsp_java.lua

```diff
@@ -0,0 +1,45 @@
+-- mod-version:3 -- lite-xl 2.1
+
+local core = require "core"
+local common = require "core.common"
+local config = require "core.config"
+local lsp = require "plugins.lsp"
+
+local installed_path_plugin = USERDIR .. PATHSEP .. "plugins" .. PATHSEP .. "lsp_java"
+local ok, jdk_info = pcall(require, "libraries.jdk")
```

Comment:
No need to `pcall` this, as it's not an optional dependency.
```suggestion
local jdk_info = require "libraries.jdk"
```